examples/pubsub/sks/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 ########################################
 # PubSub Security Key Service Examples #
 ########################################
-
-add_example(server_pubsub_central_sks server_pubsub_central_sks.c)
+if(UA_NAMESPACE_ZERO STREQUAL "FULL")
+    add_example(server_pubsub_central_sks server_pubsub_central_sks.c)  
+endif()
```

Comment:
central sks example only works with the full ns0. Should we add the full namespace to the linux build workflow or will this take too much time?

Reply:
Try it out.